src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1896,5 +1898,18 @@ private IOperation CreateDiscardExpressionOperation(BoundDiscardExpression bound
                                         ConvertToOptional(boundNode.ConstantValue),
                                         isImplicit: boundNode.WasCompilerGenerated);
         }
+
+        private IOperation CreateRangeExpressionOperation(BoundRangeExpression boundRange)
+        {
+            bool isLifted = boundRange.Type.IsNullableType();
+            return new RangeOperation(
```

Comment:
>RangeOperation [](start = 23, length = 14)

I think we should have a lazy flavor of the class here. Otherwise, we are realizing all children on the spot, that could be a lot and unnecessary.